invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteButton.js

```diff
@@ -9,43 +9,78 @@
 
 import { i18next } from "@translations/invenio_communities/i18next";
 import React, { useEffect, useState } from "react";
-import { Button, Icon, Modal } from "semantic-ui-react";
+import { Button, Icon, Modal, Message, Grid, Checkbox, Input } from "semantic-ui-react";
 import PropTypes from "prop-types";
+import { Trans } from "react-i18next";
+import { communityErrorSerializer } from "../../api/serializers";
+import { ErrorMessage } from "react-invenio-forms";
 
 export const DeleteButton = (props) => {
   const [modalOpen, setModalOpen] = useState(false);
   const [loading, setLoading] = useState(false);
-  const cancelBtnRef = React.createRef();
+  const [isChecked1, setIsChecked1] = useState(false);
+  const [isChecked2, setIsChecked2] = useState(false);
+  const [isChecked3, setIsChecked3] = useState(false);
```

Comment:
these variables are not easily understandable. How do I know which checkbox does what? could we name them in more explicit way?